js_modules/dagit/packages/core/src/instance/BackfillTerminationDialog.tsx

```diff
@@ -20,7 +20,22 @@ export const BackfillTerminationDialog = ({backfill, onClose, onComplete}: Props
     CANCEL_BACKFILL_MUTATION,
   );
   const [isSubmitting, setIsSubmitting] = React.useState(false);
-
+  const unfinishedMap = React.useMemo(() => {
+    if (!backfill) {
+      return {};
+    }
+    const unfinishedPartitions = backfill.partitionStatuses.results.filter(
+      (partition) =>
+        partition.runStatus && partition.runId && cancelableStatuses.has(partition.runStatus),
+    );
+    return (
+      unfinishedPartitions?.reduce(
```

Comment:
If you want, I think you can remove this `?` since the line above will always return an array